mxd/backend-service/src/main/java/org/eclipse/mxd/service/Impl/TransferServiceImpl.java

```diff
@@ -0,0 +1,154 @@
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.ws.rs.core.Response;
+
+import java.util.logging.Logger;
+
+
+@ApplicationScoped
+public class TransferServiceImpl implements TransferService {
+
+    private static final Logger logger = Logger.getLogger(TransferServiceImpl.class.getName());
+
+    private final TransferRepository transferRepository = new TransferRepositoryImpl();
+
+    @Override
+    public Response acceptTransfer(String transferRequest) {
+        ObjectMapper objectMapper = new ObjectMapper();
+        try {
+            TransferRequest transferRequestData = objectMapper.readValue(transferRequest, TransferRequest.class);
+            String assetsUrl = HttpServiceConnection.getUrlAssets(transferRequestData);
+            Long id = this.transferRepository.createTransferWithID(new ObjectMapper().writeValueAsString(transferRequestData), assetsUrl,
+                    transferRequestData.getId());
+            if (id != -1) {
+                TransferModelsResponse transferModelsResponse = new TransferModelsResponse();
+                transferModelsResponse.setAsset(new ObjectMapper().readTree(transferRequest));
+                return Response.ok(new ObjectMapper().writeValueAsString(transferModelsResponse.getAsset())).build();
+            } else {
+                ErrorModel errorModel = new ErrorModel(400, "Resource Not Created", "Requested Resource Not Created");
+                return Response.status(Response.Status.BAD_REQUEST)
+                        .entity(new ObjectMapper().writeValueAsString(errorModel))
+                        .type(MediaType.APPLICATION_JSON)
+                        .build();
+            }
+        } catch (Exception e) {
+            try {
+                logger.info(e.getMessage());
+                ErrorModel errorModel = new ErrorModel(500, "Internal Server Error", e.getMessage());
+                return Response.status(Response.Status.INTERNAL_SERVER_ERROR)
+                        .entity(new ObjectMapper().writeValueAsString(errorModel))
+                        .type(MediaType.APPLICATION_JSON)
+                        .build();
+            } catch (Exception ex) {
+                logger.info(ex.getMessage());
+                return Response.status(Response.Status.BAD_REQUEST)
+                        .entity("Internal Server Error")
+                        .type(MediaType.APPLICATION_JSON)
+                        .build();
+            }
+        }
+
+    }
+
+    @Override
+    public Response getTransfer(String id) {
+        try {
+            TransfersModel transfersModel = this.transferRepository.getTransferById(id);
+            if (transfersModel != null && transfersModel.getId() != null) {
+                JsonNode jsonNode = new ObjectMapper().readTree(transfersModel.getAsset());
+                TransferModelsResponse transferModelsResponse = new TransferModelsResponse();
+                transferModelsResponse.setAsset(jsonNode);
```

Comment:
There's no need of `TransferModelsResponse` class. You can directly use jsonNode.

Reply:
Now jsonNode is used directly.